### DISCUSSION TITLE
Opening a support ticket on a Hobby account (related to abuse) leads to a broken redirect

### DISCUSSION THREAD

**Discussion by AaronDewes** [vercel/community#3415] [Bug Report]
### Page to Investigate

https://vercel.com/support/cases/00148696

### Steps to Reproduce

1. Submit a support ticket.
2. Automatically get redirected to `https://vercel.com/support/cases/00148696` (Id may be different)
3. Vercel tries to automatically load a standard Deployment page, then fails and shows a 404.

**Comment by amyegan** [comment]
Hey @AaronDewes. Thank you for letting us know about this. I checked the case number and confirmed that the ticket was created successfully, but I'll follow up with the team about this 404 redirect.

**Comment by amyegan** [comment]
We shipped a fix for this bug. Thanks again for reporting it 😄

**Reply by AaronDewes** [reply]
Still seeing a 404 here (Again, deployment page loading, then 404)... Is it not deployed or did you only change the redirect, and the old link is still broken? It's not an important issue to me, so no rush :sweat_smile: 

![grafik](https://github.com/vercel/community/assets/67546953/96613414-9d4a-49b1-90d9-dfee459836b7)

**Reply by amyegan** [reply]
The confirmation page redirect was an incorrect link. We corrected the redirect for new cases. In either scenario, case updates are delivered via email 🙇
